app/views/admin/adjustments/index.html.erb

```diff
@@ -42,3 +41,19 @@
     </table>
   </div>
 </div>
+
+<script>
+  $(document).ready(function () {
+      // will call refreshPartial every 15 seconds
+      setInterval(refreshPartial, 15000)
+
+  });
+
+  // calls action refreshing the partial
+  function refreshPartial() {
+    $.ajax({
+      url: '/admin/voteringar/refresh',
+      method: 'POST'
+   })
+  }
+</script>
```

Comment:
Yeah. I did read about this, it is because of the Turbolinks.
http://product.reverb.com/2015/04/09/fun-with-setinterval-and-turbolinks/

Can try to fix it 😄